search/client.go

```diff
@@ -110,3 +109,25 @@ func (c *client) ListFacetValues(field string, maximumNumberOfValues int) (*Face
 	err = json.NewDecoder(resp.Body).Decode(facetValues)
 	return facetValues, nil
 }
+
+func createRequest(client *client, reqType string, queryStringParams map[string]string, customHeaders map[string]string, endpoint string, body io.Reader) (*http.Request, error) {
+	req, err := http.NewRequest(reqType, endpoint, body)
+	if err != nil {
+		return nil, err
+	}
+
+	customHeaders["Authorization"] = "Bearer " + client.token
+	customHeaders["User-Agent"] = client.useragent
+	for key, value := range customHeaders {
+		req.Header.Add(key, value)
+	}
+
+	queryStringParams["organizationId"] = client.orgId
```

Comment:
This could be a function `addQueryStringParam(req *http.Request, key, value string) (err, *http.Request)`

Reply:
Also be careful here, organizationId can still be null for cloudV1 no?